### PR TITLE
Merge embulk-guess-gzip

### PR DESCRIPTION
**Pull request by dmikurube**
Like we did in https://github.com/embulk/embulk-parser-csv/pull/3 and https://github.com/embulk/embulk-parser-json/pull/5, it'd be better to have the guess plugin with the corresponding parser/decoder plugin.

### CODE REVIEW

File: embulk-guess-gzip/build.gradle [hunk 1]
```diff
@@ -0,0 +1,160 @@
+plugins {
+    id "java"
+    id "maven-publish"
+    id "signing"
+    id "org.embulk.embulk-plugins" version "0.5.5"
```
Comment (hiroyuki-sato) [inline embulk-guess-gzip/build.gradle:5]: `0.5.5` is intentional? (not `0.6.1`)?

**Review by hiroyuki-sato — APPROVED** [review]
LGTM👍 (I left one comment)

File: embulk-guess-gzip/build.gradle (hunk 1 above)
Comment (dmikurube) [reply embulk-guess-gzip/build.gradle:5]: Same as: https://github.com/embulk/embulk-decoder-bzip2/pull/3#discussion_r1347412132

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**